src/Controller/ArticlesController.php

```diff
@@ -262,10 +262,12 @@ private function createFirstPage(Request $request, string $id, array $arguments)
                 return $context;
             });
 
-        $arguments['body'] = all(['item' => $arguments['item'], 'history' => $arguments['history'], 'citations' => $arguments['citations'], 'downloads' => $arguments['downloads'], 'pageViews' => $arguments['pageViews'], 'data' => $arguments['hasData'], 'context' => $context])
+        $arguments['body'] = all(['item' => $arguments['item'], 'isMagazine' => $arguments['isMagazine'], 'history' => $arguments['history'], 'citations' => $arguments['citations'], 'downloads' => $arguments['downloads'], 'pageViews' => $arguments['pageViews'], 'data' => $arguments['hasData'], 'context' => $context])
             ->then(function (array $parts) use ($context) {
```

Comment:
Not related to the change, but is $context needed to be closed over given it's passed on line 265 and reassigned on 282?